src/AppInstallerCLICore/Workflows/SourceFlow.cpp

```diff
@@ -258,7 +258,7 @@ namespace AppInstaller::CLI::Workflow
 
         for (const auto& source : sources)
         {
-            context.Reporter.Info() << Resource::String::SourceResetOne << ' ' << source.Name << "..."_liv;
+            context.Reporter.Info() << Resource::String::SourceResetOne(Utility::LocIndView{ source.Name });

```

Comment:
Good point, the reason I did not change certain field/function types (including this suggestion) is because those updates will trigger much more changes in many other files which I am trying to avoid doing in a single PR. That said, I am tracking internally suggestions and opportunities for enhancements which would be considered in separate PRs.